specification/authorization/resource-manager/Microsoft.Authorization/preview/2024-02-01-preview/RoleManagementPolicyAssignment.json

```diff
@@ -0,0 +1,393 @@
+      "get": {
+        "tags": [
+          "roleManagementPolicyAssignments"
+        ],
+        "operationId": "RoleManagementPolicyAssignments_Get",
+        "description": "Get the specified role management policy assignment for a resource scope",
+        "parameters": [
+          {
+            "name": "scope",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The scope of the role management policy.",
+            "x-ms-skip-url-encoding": true
+          },
+          {
+            "name": "roleManagementPolicyAssignmentName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of format {guid_guid} the role management policy assignment to get."
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns information about the role management policy.",
+            "schema": {
+              "$ref": "#/definitions/RoleManagementPolicyAssignment"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GetConfigurations": {
+            "$ref": "./examples/GetRoleManagementPolicyAssignmentByName.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "roleManagementPolicyAssignments"
+        ],
+        "operationId": "RoleManagementPolicyAssignments_Create",
+        "description": "Create a role management policy assignment",
+        "parameters": [
+          {
+            "name": "scope",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The scope of the role management policy assignment to upsert.",
+            "x-ms-skip-url-encoding": true
+          },
+          {
+            "name": "roleManagementPolicyAssignmentName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of format {guid_guid} the role management policy assignment to upsert."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/RoleManagementPolicyAssignment"
+            },
+            "description": "Parameters for the role management policy assignment."
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "201": {
```

Comment:
>201 [](http://example.com/codeflow?start=11&length=3)

[ARMBlockingComment] 201 is for async response. You either need to add x-ms-long-running-operation for async, or change this to 200 for sync.